indico/modules/rb_new/client/js/modules/bookRoom/filters/DateForm.jsx

```diff
@@ -17,18 +17,13 @@
 
 import moment from 'moment';
 import React from 'react';
-import RangeCalendar from 'rc-calendar/lib/RangeCalendar';
-import RCCalendar from 'rc-calendar';
 import PropTypes from 'prop-types';
-import 'rc-calendar/assets/index.css';
+import CalendarSinglePicker from 'indico/react/components/CalendarSingleDatePicker.jsx';
+import CalendarRangePicker from 'indico/react/components/CalendarRangeDatePicker.jsx';
```

Comment:
why `.jsx` here? thar's implied unless there are multiple files with the same extension (which i hope there aren't)

Reply:
also, you can import both components in one statement from `indico/react/components` if you export them from that folder's index.js